examples/currency-transfer.yaml

```diff
@@ -0,0 +1,177 @@
+'@graph':
+
+  # accounts before
+
+  - '@id': abank:b75d8f6a-e2df-4e52-b36d-1a22a66f4ead
+    '@type': EconomicResource
+    skos:note: shared account Alice has some credits on
+    classifiedAs: wd:Q179941
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 200
+    share:
+      - owner: https://alice.example/
+        quantity:
+          qudt:unit: unit:Number
+          qudt:numericValue: 150
+      - owner: https://zoe.example/
+        quantity:
+          qudt:unit: unit:Number
+          qudt:numericValue: 50
+
+  - '@id': bbank:3be5259d-10f0-431c-9fec-9c0c15a461d3
+    '@type': EconomicResource
+    skos:note: shared account Bob has some credits on
+    classifiedAs: wd:Q179941
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 500
+    share:
+      - owner: https://bob.example/
+        quantity:
+          qudt:unit: unit:Number
+          qudt:numericValue: 200
+      - owner: https://sam.example/
+        quantity:
+          qudt:unit: unit:Number
+          qudt:numericValue: 300
+
+  - '@id': abank:e0caf1e3-4a9b-4927-a19c-3f681d234529
+    '@type': Process
+    skos:note: debit Alice's credits on account
+
+  - '@id': abank:0468e9b9-4d54-4eee-9d3f-bc76db11710a
+    '@type': EconomicEvent
+    action: split
+    inputOf: abank:e0caf1e3-4a9b-4927-a19c-3f681d234529
+    resourceInventoriedAs: abank:b75d8f6a-e2df-4e52-b36d-1a22a66f4ead
+    provider: https://alice.example/
+    recipient: https://alice.example/
+    actsOnQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+
+  - '@id': abank:58c36336-09fa-4be2-b1ac-d26ea91bd982
+    '@type': EconomicEvent
+    action: merge
+    outputOf: abank:e0caf1e3-4a9b-4927-a19c-3f681d234529
+    resourceInventoriedAs: urn:uuid:9f5181db-7bd2-4ddf-92ff-10f0e8fad844
+    provider: https://alice.example/
+    recipient: https://alice.example/
+    actsOnQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+
+  - '@id': abank:b75d8f6a-e2df-4e52-b36d-1a22a66f4ead
+    '@type': EconomicResource
+    skos:note: Alice's account 
+    classifiedAs: wd:Q179941
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+    share:
+      - owner: https://alice.example/
+        quantity:
+          qudt:unit: unit:Number
+          qudt:numericValue: 150
+      - owner: https://zoe.example/
+        quantity:
+          qudt:unit: unit:Number
+          qudt:numericValue: 50
+
+  - '@id': urn:uuid:9f5181db-7bd2-4ddf-92ff-10f0e8fad844
+    '@type': EconomicResource
+    skos:note: credits in transit
+    classifiedAs: wd:Q179941
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+    share:
+      - owner: https://alice.example/
+        quantity:
+          qudt:unit: unit:Number
+          qudt:numericValue: 100
+
+  # actual ownership transfer happens only in this flow
+  - '@id': notary:da8f3360-3a6d-4ea8-b57f-4d4236fe2c3e
```

Comment:
Who is the notary in a real life example related to bank transfers?  Or in your example who would the notary be?

Reply:
I used 3rd party trusted notary so that we don't need to worry if transfer flow should have IRI in `alice:` or `bob:` namespace. Both of those personal namespaces will still have their digital record describing that *own* flow  - economic event. So notary has a copy as well as alice and bob have their copies.

Reply:
But is there any such agent in real life in currency transfers?  I don't know of any.  Hard to figure out what is happening if it doesn't match any real life observations.  Is it just to support the urn:uuid?

Reply:
I don't know details of how mainstream banks do clearings between themselves. I wouldn't expect those banks to adopt VF, they would probably just do the part of moving credits between accounts and VF records would get created outside of that banking system.
Instead of `notary:` namespace in this example I could use `urn:uuid` IRI to denote transfer flow as well. Which doesn't change that both datasets hosted on `alice:` and `bob:` namespaces can have their own digital resource which describes that flow.